lib/scripts/detect_buildpack.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2.0 License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2022-Present Datadog, Inc.
+BUILD_DIR="${BUILD_DIR:-/tmp/app}"
+DD_DETECTED_BUILDPACK=""
+if [ -f "${BUILD_DIR}"/staging_info.yml ]; then
+    DD_DETECTED_BUILDPACK=$(cat "${BUILD_DIR}"/staging_info.yml | jq '.detected_buildpack')
+elif [ -f "${BUILD_DIR}"/package-lock.json ] || [ -f "${BUILD_DIR}"/yarn.lock ]; then
```

Comment:
```suggestion
elif [ -f "${BUILD_DIR}"/app/package-lock.json ] || [ -f "${BUILD_DIR}"/app/yarn.lock ]; then
```
I am not sure if we should go with this approach, but if we do, we'll also have to check for `package.json` or `node_modules`.